docs/book/src/topics/capibmadm/powervs/port.md

```diff
@@ -0,0 +1,22 @@
+## Power VS Network Commands
+
+### 1. capibmadm powervs port list
+
+#### Usage: 
+List Power VS ports.
+
+#### Environmental Variable:
+IBMCLOUD_API_KEY: IBM Cloud api key.
+
+#### Arguments:
+--service-instance-id: Power VS service instance id.
+
+--zone: Power VS zone
```

Comment:
Needs a period at the end of argument description.
```suggestion
--zone: Power VS zone.
```

https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/pull/1086/files#r1100959368

Reply:
Addressed.